javascript/react-js/router.md

```diff
@@ -95,14 +95,18 @@ You should now have enough basics to get started with React routing. There are a
 <div class="lesson-content__panel" markdown="1">
 1. Go and add a few new routes to the application we created above; playing around with it is the best practice. Consider deleting it completely and rewriting it for practice.
 2. Browse a little bit through the React Router [documentation](https://reactrouter.com/docs/en/v6/getting-started/overview). You don't need to read through all of it, nor understand all of it. Just browse through the concepts we discussed here and re-read them. This is a great resource to refer back to.
-3. Watch [this](https://www.youtube.com/watch?v=Law7wfdg_ls) video for reviewing. You should already be familiar with those concepts.
+3. Watch [this](https://www.youtube.com/watch?v=Law7wfdg_ls) video for reviewing. You should already be familiar with those concepts.*
+
+  ***Note:** What we have learned in this lesson is the latest version of React Router (v6). However, in this video, an older version of React Router (v5) is used. The terminologies might be different, but the concept remains the same.
```

Comment:
We should have this note be a nested list under point 3.  It should be more consistent with the other lessons this way.

Reply:
Sorry, can you elaborate on `nested list under point 3`. Do you mean write it as a `4. Note:...`?

Reply:
I mean a bullet point nested within point 3. You can refer to the [Nested List section on our Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#nested-list-spacing) to see how it's done.

An example: 
```
3. lorem ipsum
    * lorem ipsum
```

Reply:
Thank you for the style guide. I have made the changes.